### ISSUE TITLE
New accessibility failures after PDF remediation

### ISSUE THREAD

**Issue by KaminariOS** [ASUCICREPO/PDF_Accessibility#26]
Commit: 1d88e46

<img width="1078" height="362" alt="Image" src="https://github.com/user-attachments/assets/7787902f-a08c-40cc-afde-4507f69eaab6" />

<img width="1156" height="789" alt="Image" src="https://github.com/user-attachments/assets/374bca56-7180-434b-bba8-8e4ad9fb0b29" />

**Comment by NickStees** [comment]
Haven't tried to deploy this yet, but I did use https://www.remediate-pdf.com/app to test it and I an also confirm it does not do table summaries. Which I think should be possible with AI.

**Comment by michaelck** [comment]
I've been taking a look at this as well and I suspect that it's an issue with the Adobe PDF Services API auto-tagger. In my case, it was overly aggressive with interpreting non-table content as a table and ended up incorrectly tagging the content.

I'm curious whether this is an active issue and/or whether there's a way to pitch in.
